test/stablecoin_converter.js

```diff
@@ -173,6 +175,18 @@ contract("StablecoinConverter", async (accounts) => {
 
       await truffleAssert.reverts(stablecoinConverter.addToken((await ERC20.new()).address), "Max tokens reached")
     })
+
+    it("Burn 10 OWL on Add Token", async () => {
+      const owlToken = await TokenOWL.new()
+      const owlProxy = await TokenOWLProxy.new(owlToken.address)
+      await owlProxy.setMinter(user_1)
+      await owlProxy.mintOWL(user_2, 10)
+
+      const stablecoinConverter = await StablecoinConverter.new(2, feeDenominator, owlToken.address)
+      const token = await ERC20.new()
+      await stablecoinConverter.addToken(token.address, { from: user_2 })
```

Comment:
There is no assert in this test. Should we add that the balance is reduced by 10 OWL?

Reply:
This test is broken and does not work at all. It is not ready and of course this missing assertion is part of that fact.